jmetal-core/src/main/java/org/uma/jmetal/util/artificialdecisionmaker/DecisionTreeEstimator.java

```diff
@@ -18,8 +18,10 @@
 public class DecisionTreeEstimator<S extends Solution<?>> {
 
   private List<S> solutionList = null;
-
-
+  private static final String VALUE_STRING = "value_";
+  private static final String NOMINAL_STRING = "my-nominal";
+  private static final String MY_STRING = "my-string";
```

Comment:
I like these ones. They have a rather clear semantic and makes it easy to create parameters for customization as a next step.